website/src/css/custom.css

```diff
@@ -2087,6 +2087,42 @@ section>h2:not(.resource-section) {
   }
 }
 
+.home-link-grid {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 24px;
+  margin-top: 4rem;
+}
+
+.home-link-grid-item {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+.home-link-grid h4 {
+  margin-bottom: .5rem;
+}
+
+.home-link-grid-item a {
+  text-decoration: none;
+  color: var(--text-neutral-primary);
+}
+
+[data-theme='dark'] .home-link-grid-item a {
+  color: var(--text-inverse-primary);
+}
+
+.home-link-grid-item a:hover {
+  text-decoration: underline;
+}
+
+@media (max-width: 996px) {
+  .home-link-grid {
+    grid-template-columns: 1fr;
+  }
+}
```

Comment:
What do you think of having them go to 2 columns at 996px, and 1 column at a smaller screen size, maybe 768px? There's a good amount of space once they drop down to 1 column initially
![image](https://github.com/user-attachments/assets/9a1de29e-68ad-4b15-861a-4fb6b3729707)

Reply:
Yeah that's a good call. Just pushed an update for 2 columns at 996px and 1 up to 768px. Looks much better.
![image](https://github.com/user-attachments/assets/53ff2959-9bd1-408d-99e1-e77dd27f46d8)